profile/index.html

```diff
@@ -95,6 +95,14 @@ <h3>Dark Mode</h3>
                 alt="bookmark"
             /></a>
           </li>
+          <li>
+            <a href="/form/index.html" area-label="Form page"
```

Comment:
Does it work with `href="/form"` as well?
```suggestion
            <a href="/form/index.html" aria-label="Form page"
```